go/vt/orchestrator/inst/instance_dao.go

```diff
@@ -2274,57 +2274,57 @@ func mkInsertOdkuForInstances(instances []*Instance, instanceWasActuallyFound bo
 		"binlog_format",
 		"binlog_row_image",
 		"log_bin",
-		"log_slave_updates",
+		"log_replica_updates",
 		"binary_log_file",
 		"binary_log_pos",
-		"master_host",
-		"master_port",
-		"slave_sql_running",
-		"slave_io_running",
+		"primary_host",
+		"primary_port",
+		"replica_sql_running",
+		"replica_io_running",
 		"replication_sql_thread_state",
 		"replication_io_thread_state",
 		"has_replication_filters",
 		"supports_oracle_gtid",
 		"oracle_gtid",
-		"master_uuid",
+		"replica_uuid",
```

Comment:
:exclamation: :exclamation: :exclamation: this should be `primary_uuid` or `source_uuid`, not `replica_uuid`

Reply:
changed